_docs/ios/1.207.0.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: 1.207.0
+permalink: /docs/ios/1.207.0/
+parent: iOS
+nav_order: -207
+release_date: 15.10.2024
+---
+
+## Lock accessories now display the device name in the navigation bar instead of the page title.
+When you enter settings page of Lock accessory, you will see device name in navigation bar instead of page title. This also applied to "Users" and "Access" pages of this device.\
+![screenshot](/tedee-release-notes/docs/ios/assets/1.207.0-navigation-bar-device-name.png)
+
+## Improved app performance.
+Application should open quicker for the user with a lot of devices on their account. Also, refreshing of my devices list should be smoother.
+
+## Preview: Added a new quick configuration sharing option in the device addition process.
+We updated the design and functionality of quick configuration sharing option in the device addition process. This update provide not only new designs but also, possibility to select known user, phone contact or organization user from the list instead of need to provide full email address every time. Possibility to provide new user via email address is still available.\
```

Comment:
We have updated

generally more appriopriate form is `We have did something` also in next points